### PR TITLE
add data.js v0.3.0 with autoupdate config added

### LINKED ISSUE #7052: [Request] Add data.js
**Library name:** data.js
**Git repository url:** https://github.com/yanhaijing/data.js
**npm package url(optional):** data_js

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #7052 
@PeterDaveHello please help me review the pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/yanhaijing/data.js
 * Official website (optional, not the repository): 
 * NPM package url (optional): https://www.npmjs.com/package/data_js
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 259
   - Count of watchers: 27
   - Count of forks: 59
 * NPM download stats (optional):
   - Downloads in the last day: 0
   - Downloads in the last week: 16
   - Downloads in the last month: 64

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7052, cc @yanhaijing

### CODE REVIEW

File: ajax/libs/data.js/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "type": "git",
+    "url": "git://github.com/yanhaijing/data.js.git"
+  },
+  "license": "MIT",
+  "keywords": [
+    "data",
+    "data.js",
+    "Data"
+  ],
+  "gitHead": "3e5bfe1a1bb1886097a8c6d76e50f22b14a75578",
```
Comment (PeterDaveHello) [inline ajax/libs/data.js/package.json]: Please remove `gitHead` section, thanks.

File: ajax/libs/data.js/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "data.js",
+  "filename": "data.min.js",
+  "description": "data.js 是带有消息通知的数据中心，我称其为会说话的数据。旨在让编程变得简单，世界变得美好。",
```
Comment (PeterDaveHello) [inline ajax/libs/data.js/package.json]: Would you please use English here?

File: ajax/libs/data.js/package.json (hunk 1 above)
Comment (pvnr0082t) [inline ajax/libs/data.js/package.json]: Okay, I will fix it ASAP. Thanks

File: ajax/libs/data.js/package.json (hunk 2 above)
Comment (pvnr0082t) [inline ajax/libs/data.js/package.json]: OK, I will translate later. Thank you.

**Comment by pvnr0082t** [comment]
@PeterDaveHello I have updated the ```package.json``` file.
Could you please help me review again? Thank you.

**Comment by PeterDaveHello** [comment]
Okay, please squash these two commits into only one commit, so that I can merge it, thanks.

**Comment by pvnr0082t** [comment]
OK, I have squash the commit into one. Thanks

**Pull request merged**